tests/framework/extensions/kubectl/explain.go

```diff
@@ -0,0 +1,65 @@
+package kubectl
+
+import (
+	"fmt"
+	"strings"
+
+	apisV1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/kubeconfig"
+	"github.com/rancher/rancher/tests/framework/extensions/workloads"
+	"github.com/rancher/rancher/tests/framework/extensions/workloads/pods"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// Explain is a helper function that creates a Job by calling the helper CreateJobAndRunKubectlCommands and executing kubectl explain in the pod of the Job
+//returns the output from the pod logs.
```

Comment:
need to fix this lint mistake, you need a space here.

Reply:
Also public_api is not a proper directory name. It should be publicapi as one name